src/util/__snapshots__/index.test.js.snap

```diff
@@ -2,10 +2,10 @@
 
 exports[`formatDatePair displays differences of dates 1`] = `"Mon, Feb 4, 2019 at 6:41PM GMT"`;
 
-exports[`formatDatePair displays differences of dates 2`] = `"Mon, Feb 4, 2019 at 6:41PM GMT - 9:41PM GMT"`;
+exports[`formatDatePair displays differences of dates 2`] = `"Mon, Feb 4, 2019 at 6:41PM GMT - Mon, Feb 4 at 9:41PM GMT"`;
```

Comment:
This is the result of simplifying the end date logic in formatDatePair. It will always render a complete date.

Reply:
not totally convinced this is better, but im open to convincing :) to me if the date stays the same then you just want to know the time, not display the same date twice

Reply:
My intent here was to provide clarity as the formatting didn't make it obvious to me that the event was starting and ending on the same day. However, if the preference is to revert it to the current formatting I can push a fix to this PR. No problem :)

Reply:
Really what it should be if the event starts and ends on the same is like this: Mon, Feb 4, 2019 at 6:41PM - 9:41PM GMT. So back to the way it was but only show the time zone once. This makes it more obvious that its a time range on the same day. Let's do that. Thanks!

Reply:
I made the changes requested, let me know what you think :)